### PR TITLE
[YANG] Enable Yang model for BGP_BBR config entry

### PR DESCRIPTION
**Pull request by Gfrom2016**
#### Why I did it
Enable Yang model for BGP_BBR config entry.
```
{
        "BGP_BBR": {
            "all": {
                "status": "enabled"/"disabled"
            }
        }
}
```
##### Work item tracking
- Microsoft ADO **(number only)**: 25988660

#### How I did it
Add yang model and ut for BGP_BBR.

#### How to verify it
Use GCU cmd to change bbr status.
Create following json patch: disable_bbr.json-patch
```
[
 {
  "op": "replace",
  "path": "/BGP_BBR/all/status",
  "value": "disabled"
 }
]
```
Run ```sudo config apply-patch ./disable_bbr.json-patch``` cmd on dut. Success.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205
- [x] 202211
- [x] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: src/sonic-yang-models/yang-models/sonic-bgp-bbr.yang [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    container sonic-bgp-bbr {
+        container BGP_BBR {
+
+            description "BGP_BBR table part of config_db.json";
+
+            container all {
+                leaf status {
+                    type string {
+                        pattern "enabled|disabled";
```
Comment (wen587) [inline src/sonic-yang-models/yang-models/sonic-bgp-bbr.yang:28]: We can use admin_mode for this :
https://github.com/sonic-net/sonic-buildimage/blob/d436dd6d84db05b7a1833bb5a1482a8763afcd3b/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L127

Comment (wen587) [reply src/sonic-yang-models/yang-models/sonic-bgp-bbr.yang:28]: Others lgtm

**Comment by qiluo-msft** [comment]
@ganglyu Could you also help review?

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**